### PR TITLE
Mbxfxp 34 - transfers

### PR DESCRIPTION
**Pull request by ramirogm**

### CODE REVIEW

File: src/lib/model/lib/requests/backendRequests.js [hunk 1]
```diff
@@ -130,7 +130,67 @@ class BackendRequests {
...
+     * @param {String} sourceFSP
+     * @param {String} destinationFSP
+     */
+    async postFxpTransferResponse(transferId, fxpTransferResponse) {
+        // MOCKING until backend ready
+
+        //return this._post(`fxptransfers/${transferId}/responses`, fxpTransferResponse);
+        // mocking
+        // no headers returned
+        let mockedResponse = {
```
Comment (chinogarcia) [inline src/lib/model/lib/requests/backendRequests.js]: now it will return no-content

**Review by chinogarcia — CHANGES_REQUESTED** [review]
Seems ok except the mock part, to validate I need to test it with the FXP PR

File: src/lib/model/lib/requests/backendRequests.js (hunk 1 above)
Comment (ramirogm) [reply src/lib/model/lib/requests/backendRequests.js]: OK

Comment (ramirogm) [reply src/lib/model/lib/requests/backendRequests.js]: fixed

**Review by ramirogm — COMMENTED** [review]
Ok thanks

**Pull request merged**